.eslintrc.js

```diff
@@ -1,10 +1,24 @@
-/**
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
-};
\ No newline at end of file
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'airbnb',
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+  },
```

Comment:
@harrisonywu @krise3 I've seen this configuration in another student's repo as well. I'm wondering where it came from because I believe much of it is unnecessary (the globals "Atomics" or "SharedArrayBuffer", the whole "parserOptions" chunk, and the "plugins" chunk).